test/datalog_tests/hashset_test.dump.expected

```diff
@@ -0,0 +1,10 @@
+hashset_test::SetTransforms{.description = "all(contains(3))", .s = []}
+hashset_test::SetTransforms{.description = "any(contains(3))", .s = [(2, [4, 5, 6]), (4, [8, 9, 10]), (3, [6, 7, 8]), (1, [2, 3, 4]), (0, [0, 1, 2])]}
+hashset_test::SetTransforms{.description = "arg_max(max)", .s = [(4, [8, 9, 10])]}
+hashset_test::SetTransforms{.description = "arg_min(min)", .s = [(0, [0, 1, 2])]}
+hashset_test::SetTransforms{.description = "filter(contains 2)", .s = [(1, [2, 3, 4]), (0, [0, 1, 2])]}
+hashset_test::SetTransforms{.description = "filter_map(==1, :=100)", .s = [(100, [2, 3, 4])]}
+hashset_test::SetTransforms{.description = "find(contains(3))", .s = [(1, [2, 3, 4])]}
+hashset_test::SetTransforms{.description = "map(push 100)", .s = [(1, [2, 3, 4, 100]), (3, [6, 7, 8, 100]), (2, [4, 5, 6, 100]), (0, [0, 1, 2, 100]), (4, [8, 9, 10, 100])]}
+hashset_test::SetFolds{.description = "fold(+)", .a = 10}
+hashset_test::SetFolds{.description = "fold(fold(+))", .a = 75}
```

Comment:
This makes me wonder if we shouldn't make `Collection::assert_eq()` a ddlog expression (similar to how `apply` works)

Reply:
Can you elaborate?  How would this work?  This would have to be part of the command language, not DDlog itself, right?

Reply:
Basically I just see it taking advantage of the fact that vectors implement `.to_stream()`. You'd make a vec of the contents you expected to see and then we'd call ddflow's `assert_eq()` on the collection of expected items and the relation you want to check. Admittedly this only has fairly limited use since the user would have to specify timestamps, but I think it could still be genuinely useful. In a similar vein `.assert_empty()` could also be useful